CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The minor version will be incremented upon a breaking change and the patch versi
 ### Fixes
 
 - idl: Update `proc-macro2` usage for latest nightly ([#3663](https://github.com/solana-foundation/anchor/pull/3663))
+- ts: Fix parsing idl with multiple const generics ([#3665](https://github.com/solana-foundation/anchor/pull/3665))
```

Comment:
There is no option to run the CI tests after the merge commit for some reason, so I'm making a non-important change to trigger CI here.
```suggestion
- ts: Fix parsing IDL with multiple const generics ([#3665](https://github.com/solana-foundation/anchor/pull/3665))
```